DOL.WHD.Section14c.Web/src/modules/components/sectionWioa/sectionWioaTemplate.html

```diff
@@ -14,122 +14,132 @@
   </div>
 </div>
 
-<div class="form-page">
-  <h3>Workforce Innovation and Opportunity Act (WIOA)</h3>
-  <hr>
-  <p>WIOA requirements are effective July 22, 2016. <a href ng-click="vm.toggleLearnMore()">Learn More</a></p>
```

Comment:
Since `vm.toggleLearnMore()` isn't being used anymore, should it be removed from `sectionWioaController.js` too?  There are a couple of tests in `sectionWioaController.test.js` that could come out in that case as well.

Reply:
Hm, maybe a question for @davoodharun ?  I'm applying the new help button patterns page by page, so not sure where the old code should come out. I'm all for removing unused code!